cypress/integration/host/host_events.ts

```diff
@@ -206,8 +206,9 @@ describe("Host events", () => {
 
   it("host events are displayed in the right timezone", () => {
     cy.visit("/preferences");
-    cy.dataCy("timezone-field").click();
-    cy.dataCy("Hawaii-option").click();
+    cy.get('[data-testid="leafygreen-ui-select-menubutton"]').first().click();
+    cy.get('[data-leafygreen-ui="option"]').eq(3).contains("Hawaii");
+    cy.get('[data-leafygreen-ui="option"]').eq(3).click();
     cy.dataCy("save-profile-changes-button").click();
 
     cy.visit(pathWithEvents);
```

Comment:
The underlying code for `leafygreen-ui` Select doesn't actually use `<select>`, which is why I did some less than ideal stuff here 😔 LMK if you have better ideas